src/window.rs

```diff
@@ -708,7 +708,7 @@ impl Window {
     /// - **Wayland / X11:** Not implemented.
```

Comment:
Could add a note about `None` that it means that visibility can't be determined? Also you should likely add a note that it's not recommended way to drive your rendering backend, etc, etc.

Also, on Wayland it's `Unsupported` and not `Not implemented`, since it can't be implemented.